docs/pages/versions/v36.0.0/workflow/configuration.md

```diff
@@ -261,7 +261,7 @@ Configuration for remote (push) notifications.
       Local path or remote url to an image to use as the icon for push notifications.
```

Comment:
```suggestion
      Local path or remote url to an image to use as the icon for push notifications (Android only)
```

Reply:
great!

Reply:
I'm still testing this change on android, so far it doesn't seem to be working ☹️